### PR TITLE
[DUOS][risk=no] translate page x2 - added error handling, added no results block, updated url to be hardcoded

### PR DESCRIPTION
**Pull request by lu-c**
The url is now up and running (despite CORS error), so had to make a few tweaks to this code

----
Have you read [Terra's Contributing Guide](https://github.com/DataBiosphere/terra-ui/wiki/Contributor-Guide) lately? If not, do that first.

- Label PR with a Jira ticket number and include a link to the ticket
- Label PR with a security risk modifier [no, low, medium, high]
- PR describes scope of changes
- Get a minimum of one thumbs worth of review, preferably two if enough team members are available
- Get PO sign-off for all non-trivial UI or workflow changes
- Verify all tests go green
- Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/libs/ajax.js [hunk 1]
```diff
@@ -880,7 +880,7 @@ export const ToS = {
 
 export const Translate = {
   translate: async (body) => {
-    const url = `${await getOntologyUrl()}/translate/paragraph`;
+    const url = `https://consent-ontology.dsde-dev.broadinstitute.org/translate/paragraph`;
```
Comment (lu-c) [inline src/libs/ajax.js]: `await getOntologyUrl()` was returning `https://consent-ontology.dsde-dev.broadinstitute.org/` (with the slash on the end of the string), but other urls in this ajax file looked like `${getOntologyUrl()}/autocomplete`, and when following that standard, the translate URL turned into

`https://consent-ontology.dsde-dev.broadinstitute.org//translate/paragraph` which would error out (because of the double slash)

Not sure if it's safe to remove that slash from the url string, and time is short for this demo, so to rush it, I just threw in the actual url as a string instead of relying on the config

**Comment by lu-c** [comment]
The actual endpoint throws a CORS error - Please correct me I'm wrong, but there's nothing we can do about that, and it might be okay in dev? But if any fixes need to happen, it's in the api?

File: src/libs/ajax.js (hunk 1 above)
Comment (rushtong) [reply src/libs/ajax.js]: Don't we need the config for local proxying?

**Comment by rushtong** [comment]
> The actual endpoint throws a CORS error - Please correct me I'm wrong, but there's nothing we can do about that, and it might be okay in dev? But if any fixes need to happen, it's in the api?

The CORS error is because we're not going through the local proxy.

File: src/pages/Translator.js [hunk 2]
```diff
@@ -133,7 +117,14 @@ export default function Translator() {
             style: { width: '100%', justifyContent: 'center' }
           }, [Spinner]),
 
-          div({ isRendered: !isLoading, className: 'row no-margin' }, [
+          div({ isRendered: error }, [
+            'There was an error running your request'
```
Comment (connorlbark) [inline src/pages/Translator.js]: do we want to use error to show something more specific

File: src/libs/ajax.js (hunk 1 above)
Comment (lu-c) [reply src/libs/ajax.js]: Do we? When I throw in this line

`const url = `${await getOntologyUrl()}translate/paragraph``

it errors on `POST https://local.broadinstitute.org:3000/translate/paragraph 404 (Not Found)`

**Review by connorlbark — COMMENTED** [review]
was able to test it ok, the changes themselves look good, once we figure out the config stuff I will retry + approve.

**Review by rushtong — APPROVED** [review]
👍🏽

File: src/pages/Translator.js (hunk 2 above)
Comment (lu-c) [reply src/pages/Translator.js]: i dont want to show the error message if it's something wrong with our servers or our code, only if it's a user error (that they can do something about). For now, I think this is okay, since I'm just trying to rush this code in real quick ^_^;

**Review by JVThomas — APPROVED** [review]
UI code is hitting the endpoint and relaying the results to the client.
